src/networking.c

```diff
@@ -3374,6 +3377,17 @@ NULL
         } else {
             addReplyArrayLen(c,0);
         }
+    } else if (!strcasecmp(c->argv[1]->ptr, "no-touch") && c->argc == 3) {
```

Comment:
Not a strong opinion, but I would expect this to be reflected in `clearClientConnectionState` as well. I'm surprised No-evict isn't in that list, might have been an oversight?

Reply:
I am not sure if we should add no-evict to that list, should we allow clients be evicted once it is reset? But I think no-touch flag should be cleared in the function.

Reply:
I agree. Let's add no-evict as well, this function should return the client to the same sate it was when first connected

Reply:
And let's mention this change (and all other interface changes) in the PR top comment (for commit comment and release notes)